data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+			`	"Go to their home system and make your request, the Hai government despises conflict, so they will most likely grant it. They won't let you land on their homeworld given the circumstances, though, so go to Mirrorlake instead."`
+				accept
+	on complete
+		conversation
+			`You find the Hai leaders that agreed to meet you and express your request to settle the disputes between you. They consider this amongst themselves before responding, "Tell us, <last>, captain of the <ship>, why should we grant you such a request?"`
+			choice
+				`	"Because otherwise you get shot!" (Draw your weapon)`
+					goto death
+				`	"I know I have done stupid things in the past, but I have learned my error. I promise not to make such mistakes again."`
+				`	"I can continue bringing food to the Unfettered for you, so they do not starve to death."`
+			`	They seem willing to give you a second chance. "You have convinced us, but we will not grant you this pardon twice."`
+				accept
+			label death
+			`	The leaders gasp, and then, in a moment that seems to last forever, there is silence whilst everyone is looking at you. The silence is shattered when the guards start shooting. They probably did not intend to kill you outright, but apparently they see you as enough of a threat that the result is the same.`
+				die
+		"reputation: Hai" >?= 1
+		log `Convinced the Hai to stop their conflict with me, as request by an Unfettered elder.`
+
+
+
+mission "Unfettered: Bring Hai"
+	name `Bring True Hai back`
+	description `Bring <bunks> Hai from <stopovers> to <origin>.`
+	source "Firelode"
+	stopover "Snowfeather"
+	passengers 10
+	blocked "You will need room for 10 passengers in order to take the next Unfettered mission."
+	to offer
+		has "Unfettered Jump Drive 1: offered"
+		or
+			not "Unfettered: offered bring hai"
+			random < 20
+	on defer
+		set "Unfettered: offered bring hai"
+	on offer
+		conversation
+			`As you exit your ship you notice a group of Unfettered that seem to be discussing something of importance in a big, yet simple, square-shape stadium nearby. A few Unfettered are already heading their way to listen, it does not seem to be restricted of access.`
+			choice
+				`	(Go see them.)`
+				`	(Ignore them.)`
+					defer
+			`	As you head towards them one separates from the rest and comes to meet you. He seems to be lost in his thought and is mumbling on, something about, "We do not get alien visitors often here. There has to be a reason for your presence..." When he speaks again, directly to you, it is with more focus, "Sorry, I was just thinking out loud! Well you are here anyway so maybe you could help us?"`
+			`	"We have friends outside of our space, civilians and others. You see, we would do anything to help a friend, and a few Hai want to join us. I can only hope you are willing to help us reunite with our brothers and sisters from Hai space," he says, apparently a bit desperate, considering he knows very little if anything at all about you.`
```

Comment:
I have no idea what this scene is actually doing, and this Hai seems very confused and not at all like they're representing the group, which I presume was the intent.

Reply:
Right, yeah none of that is clear at all.

Reply:
How does it seem now?